### PR TITLE
Make changes file reproducible

### PR DESCRIPTION
**Pull request by norrisjeremy**
Make changes file reproducible (#712)

### CODE REVIEW

File: src/main/java/org/vafer/jdeb/changes/TextfileChangesProvider.java [hunk 1]
```diff
@@ -46,15 +47,24 @@ public final class TextfileChangesProvider implements ChangesProvider {
 
     private final ChangeSet[] changeSets;
 
-    private DateFormat fmt = new SimpleDateFormat("HH:mm dd.MM.yyyy");
+    private final DateFormat fmt;
```
Comment (tcurdt) [inline src/main/java/org/vafer/jdeb/changes/TextfileChangesProvider.java:50]: Why did you move this into the constructor? Just curious.

Comment (norrisjeremy) [reply src/main/java/org/vafer/jdeb/changes/TextfileChangesProvider.java:50]: Because we conditionally call `fmt.setTimeZone(TimeZone.getTimeZone("UTC"))` depending upon whether or not the `outputTimestampMs` argument is null,  it seemed more logical to keep the initialization of `fmt` near to this call.

Additionally, since `fmt` is not mutated outside of the constructor, it seemed prudent to declare it `final` as well.

**Review by Phubao8821 — APPROVED** [review]

**Comment by tcurdt** [comment]
Looking good to me. Thanks for the contribution!
